src/main/resources/mail-templates/authorization-email.html

```
@@ -16,4 +16,5 @@
     <br/>
     <p th:text="#{email.regards}"></p>
   </body>
-</html>
\ No newline at end of file
+</html>
+</html>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </html> ]

(tag-pair)